docs/Fields.md

```diff
@@ -377,7 +377,7 @@ Then admin-on-rest renders the `<CommentList>` with a loader for the `<Reference
 
 ## `<ReferenceManyField>`
 
-This component fetches a list of referenced record (using the `GET_MANY_REFERENCE` REST method), and passes the result to an iterator component (like `<SingleFieldList>` or `<Datagrid>`). The iterator component usually has one or more child `<Field>` components.
+This component fetches a list of referenced records by filtering with a field of the current record (using the `GET_MANY_REFERENCE` REST method), and passes the result to an iterator component (like `<SingleFieldList>` or `<Datagrid>`). The iterator component usually has one or more child `<Field>` components.
```

Comment:
This is not very clear. Could you rephrase, or perhaps illustrate with an example record? Same for `ReferenceArrayField`